packages/service/lib/compile.js

```diff
@@ -48,29 +48,29 @@ async function setup (cwd, config, logger) {
   /* c8 ignore next 4 */
   if (tscExecutablePath === undefined) {
     const msg = 'The tsc executable was not found.'
-    logger.error(msg)
+    logger.warn(msg)
   }
 
-  const tsconfigPath = resolve(cwd, 'tsconfig.json')
-  const tsconfigExists = await isFileAccessible(tsconfigPath)
+  const tsConfigPath = config?.plugins?.typescript?.tsConfig || resolve(cwd, 'tsconfig.json')
```

Comment:
will this still be `cwd` relative?

Reply:
Yes, resolving this is handled by the config automatically